apps/web/app/hooks/features/useOrganizationTeams.ts

```
@@ -76,398 +72,455 @@
+	}, [loading, setTeamsFetching]);
+
+	const setActiveTeam = useCallback(
+		(team: (typeof teams)[0]) => {
+			setActiveTeamIdCookie(team?.id);
+			setOrganizationIdCookie(team?.organizationId);
+			// This must be called at the end (Update store)
+			setActiveTeamId(team?.id);
+
+			// Set Project Id to cookie
+			// TODO: Make it dynamic when we add Dropdown in Navbar
+			if (team && team?.projects && team.projects.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)